solidity/package.json

```diff
@@ -1,15 +1,19 @@
 {
 	"version": "1.0.0",
+	"type": "module",
 	"description": "Augur-Constant-Product-Market-Contracts",
 	"scripts": {
-		"compile": "tsc && node ./compile.js && node --input-type=module -e \"import { promises as fs } from 'fs'; await fs.rm('./compile.js')\""
+		"compile": "tsc && node ./js/compile.js",
+		"test": "tsc && node ./js/tests/test.js"
 	},
 	"keywords": [],
 	"author": "",
 	"license": "ISC",
 	"devDependencies": {
 		"@types/node": "22.13.10",
 		"typescript": "5.8.2",
-		"solc": "0.8.29"
+		"solc": "0.8.29",
+		"viem": "2.23.12",
```

Comment:
Why does `solidity` project need `viem`?  Is this for interacting with a blockchain via NodeJS?